modules/update-restricted.adoc

```diff
@@ -34,13 +34,23 @@ The release image signature config map allows the Cluster Version Operator (CVO)
 +
 [source,terminal]
 ----
-$ oc adm upgrade --allow-explicit-upgrade --to-image ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}@<digest> <1>
+$ oc adm upgrade --allow-explicit-upgrade --to-image <defined_registry>/<defined_repository>@<digest>
 ----
-<1> The `<digest>` value is the sha256 digest for the targeted release image, for example, `sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92`
 +
-If you use an `ImageContentSourcePolicy` for the mirror registry, you can use the canonical registry name instead of `LOCAL_REGISTRY`.
+Where:
++
+** `<defined_registry>` is the name of the mirror registry you mirrored your images to.
```

Comment:
I see there are some other `Where:` section in same page, but they have different format, shell we keep them same? Nothing else are indented. Like:
```
where:

<path_to_the_directory>
Specifies the path to the directory, if you used the --dest-dir flag.

<file_name>
Specifies the name of the ImageDigestMirrorSet YAML.
```

Reply:
Thanks for catching this!! I don't use this style much so I was just taking a guess at the formatting, but I'll follow the format that you're pointing to.